src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
```

Comment:
How about breaking this component into several components? That way we'll get a clear separation of concerns between the components.